service/control.go

```diff
@@ -69,10 +75,36 @@ func (ctrl *Control) postWorkflow(c *gin.Context) {
 		Owner:       body.Owner,
 		IsActive:    body.IsActive,
 	}
-	ctrl.db.Create(&wf)
+	// upsert workflow
```

Comment:
hmm, if we are doing upserts, we probably need to change to `PUT` instead of `POST`

Reply:
done